src/update-manager.ts

```diff
@@ -957,7 +972,8 @@ export default class UpdateManager {
   updateLocalFile (doc: NamedNode, ds, is, callbackFunction): void {
     const kb = this.store
     // console.log('Writing back to local file\n')
-    // See http://simon-jung.blogspot.com/2007/10/firefox-extension-file-io.html
+    // previously used http://simon-jung.blogspot.com/2007/10/firefox-extension-file-io.html ... now uses PUT
```

Comment:
**question:** So is this link still relevant or not? If not let's remove it, otherwise it will only lead to confusion

Reply:
No, if we remove the code for it, it is only historically relevant.

Reply:
**suggestion:** Let's remove it then. It is still present in the git history, together with the code it refers to